drivers/mdio/mdio_xilinx_axienet.c

```diff
@@ -0,0 +1,200 @@
+/*
+ * Xilinx AXI 1G / 2.5G Ethernet Subsystem
+ *
+ * Copyright(c) 2024, CISPA Helmholtz Center for Information Security
+ * SPDX - License - Identifier : Apache-2.0
+ */
+#define LOG_MODULE_NAME eth_xilinx_axienet_mdio
+#define LOG_LEVEL       CONFIG_ETHERNET_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
you can use two arguments to this macro to avoid the LOG_LEVEL #define, and dont need to define a LOG_MODULE_NAME either.

Reply:
Done